### ISSUE TITLE
GPL-905 - As a heron laboratory user I need to be able to print reagent aliquot numbers [Heron] (C=M, V=?)

### ISSUE THREAD

**Issue by stevieing** [sanger/lighthouse-ui#54]
**User story**
As a heron laboratory user I need to be able to print reagent aliquot numbers.  This should be done by adding a new page to Lighthouse UI, like https://lighthouse-ui.psd.sanger.ac.uk/print_labels but with two free text fields and a barcode option.  A quantity is also needed.

**Who are the primary contacts for this story**
Anna / Steve

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] Add a new page for printing labels for reagent aliquot numbers.
- [x] Include two free text fields and a barcode field to print on labels.
- [x] Add a quantity option to print duplicates of the label.
- [x] Limit the quantity of barcodes to 100, to prevent accidents when specifying the quantity.
- [ ] Ensure that the BC-5 printer is working correctly.
- [ ] Add BC-6 and BC-7 printers to the availability list.

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
The proposed label layout could be https://sprint.psd.sanger.ac.uk/?labelType=thin&printRequest=%7B%22layouts%22%3A%5B%7B%22textFields%22%3A%5B%7B%22fontSize%22%3A%222.3%22%2C%22font%22%3A%22proportional%22%2C%22rotation%22%3A%22north%22%2C%22value%22%3A%22First+line+of+free+text%22%2C%22x%22%3A3%2C%22y%22%3A%224%22%7D%2C%7B%22fontSize%22%3A%222.3%22%2C%22font%22%3A%22proportional%22%2C%22rotation%22%3A%22north%22%2C%22value%22%3A%22A+second+line+of+free+text%22%2C%22x%22%3A3%2C%22y%22%3A%227%22%7D%5D%2C%22barcodeFields%22%3A%5B%7B%22x%22%3A39%2C%22y%22%3A1%2C%22value%22%3A%22BATCH+225%22%2C%22cellWidth%22%3A%220.2%22%2C%22height%22%3A%224%22%2C%22rotation%22%3A%22north%22%2C%22barcodeType%22%3A%22code39%22%7D%5D%7D%5D%7D

**Comment by rl15** [comment]
This looks candidate for Mixtio.

**Comment by stevieing** [comment]
> This looks candidate for Mixtio.

Are you sure? I think they just want to print labels. Not sure if they need to track reagents.

**Comment by stevieing** [comment]
@stevieing find out whether they need this??

**Comment by sdjmchattie** [comment]
From Ollie C:

> I have printed barcodes from BC-1, BC-3 all seem fine their BC5 (or BC-10), whatever it is called now, is having a new work error please see attached.
